src/components/routes/Host/ListingForm/ListingForm.tsx

```diff
@@ -167,22 +224,34 @@ class ListingForm extends React.Component<Props, State> {
                     <Route
                       exact
                       path="/host/listings/:id/accommodations"
-                      render={() => <AccommodationsForm {...FormikProps} />}
+                      render={() => (
+                        <AccommodationsForm {...FormikProps} ListingField={ListingField} setFocus={this.handleFocus} />
```

Comment:
Passing in the `ListingField` enum as a prop here feels funny; could `ListingField` just be exported from somewhere appropriate and shared by the various form pages?

Reply:
was thinking the same. Any suggestions on where it belongs? I was thinking `networking/listings` but it might be out of scope for that file

Reply:
sgtm. this should make the form much cleaner!